src/app/components/button/button.tsx

```diff
@@ -1,13 +1,23 @@
+import { Arrow } from '@/icons/arrow';
+
 import './button.scss';
 
-type ButtonProps = {
+export type ButtonProps = {
   label: string;
   href: string;
 };
 
-export const Button = ({ label, href }: ButtonProps) => (
-  <a className="button" href={href} target="_blank" rel="noreferrer">
-    <span className="label">{label}</span>
-    <span className="material-symbols-outlined arrow">arrow_forward</span>
-  </a>
-);
+export const MakeBtnTemp = (isOutlined: boolean) => {
```

Comment:
I suppose we should create some 'universal' button. Now we have 3 (or 4) different button but what will we do if they will be 10? I suggest expand ButtonProps and create 1 component but with different pros (classes, text, borders, icons, etc.